packages/roosterjs-editor-api/lib/format/toggleBullet.ts

```diff
@@ -21,13 +25,21 @@ export function workaroundForEdge(editor: Editor): HTMLElement {
     return null;
 }
 
+/**
+ * Remove the workaroundSpan after toggling bullet in Edge
+ * @param workaroundSpan The workaround span that was added
+ */
 export function removeWorkaroundForEdge(workaroundSpan: HTMLElement) {
     if (workaroundSpan && workaroundSpan.parentNode) {
         workaroundSpan.parentNode.removeChild(workaroundSpan);
     }
 }
 
-export default function toggleBullet(editor: Editor): void {
+/**
+ * Toggle bullet at selection
+ * @param editor The editor instance
```

Comment:
explain what will happen when selection contains both bullet and normal text,
when selection contains bullet in deep level
when selection contains number list